include/cantera/kinetics/Reaction.h

```diff
@@ -286,6 +386,49 @@ std::vector<shared_ptr<Reaction> > getReactions(const XML_Node& node);
 //! Kinetics object `kinetics`.
 std::vector<shared_ptr<Reaction>> getReactions(const AnyValue& items,
                                                Kinetics& kinetics);
+
+//! Parse reaction equation
+void parseReactionEquation(Reaction& R, const AnyValue& equation,
+                           const Kinetics& kin);
+
+// declarations of setup functions
+void setupElementaryReaction(ElementaryReaction&, const XML_Node&);
+void setupElementaryReaction(ElementaryReaction&, const AnyMap&,
+                             const Kinetics&);
+
+void setupThreeBodyReaction(ThreeBodyReaction&, const XML_Node&);
+void setupThreeBodyReaction(ThreeBodyReaction&, const AnyMap&,
+                            const Kinetics&);
```

Comment:
These are just place holders to recycle existing code. For XML I wouldn't want to change anything beyond, but for YAML you are correct. It would just create a much larger diff if these were moved already.